pipeline/transformer/transformer_sierra/src/test/scala/weco/pipeline/transformer/sierra/transformers/SierraGenresTest.scala

```diff
@@ -247,23 +306,51 @@ class SierraGenresTest
         )
       )
     )
+    //TODO: This would be clearer without looping
```

Comment:
Can you use https://www.scalatest.org/user_guide/table_driven_property_checks?

Reply:
In this case, I think that just unrolling it and doing the assertions twice is the better plan.